src/pages/en/guides/markdown-content.md

```diff
@@ -35,11 +39,14 @@ To learn more about adding a layout to your page, read the next section on **Mar
 
 ### Frontmatter `layout`
 
-In Astro, Markdown and MDX pages have a special frontmatter property for `layout` that defines the relative path to an Astro [layout component](/en/core-concepts/layouts/). This component will wrap your Markdown content, providing a page shell and any other included page template elements.
+In Astro, Markdown and MDX pages have a special frontmatter property for `layout` that defines the relative path to an Astro [layout component](/en/core-concepts/layouts/#markdown-layouts). This component will wrap your Markdown content, providing a page shell and any other included page template elements.
 
-```markdown
+```markdown {6} 
 ---
 // src/pages/page.md
+title: "Astro v1 Launch!"
+author: "Matthew Phillips"
+date: "09 Aug 2022"
 layout: ../layouts/BaseLayout.astro
 ---
```

Comment:
Just a quick shout out to @kylebutts' suggestion that there could be coordination between snippets! So, I added the properties here that are being shown in the layout component below. Thanks Kyle!  (And note to self to add the co-author cred manually!)